src/sysinfo.cc

```diff
@@ -75,11 +75,12 @@ bool ReadIntFromFile(const char* file, long* value) {
     char line[1024];
     char* err;
     memset(line, '\0', sizeof(line));
-    CHECK(read(fd, line, sizeof(line) - 1));
```

Comment:
Not your problem, but it is utterly confusing that `CHECK` actually means `DCHECK`.  If we ever want to layer the benchmark library on top of glog (I am still hoping that one day all the bits of code that Google open-sources will work together) this will get in the way.